scripts/globals/job_utils/thief.lua

```diff
@@ -38,6 +38,17 @@ local stealableSPEffects =
 -----------------------------------
 -- Local Functions
 -----------------------------------
+local function getTHFlvl(player)
```

Comment:
I've seen this pattern used for several jobs, and would suggest in the future we create a generic utils function to return.  _also_ there is no safety for the subjob actually being thf here, but I would assume this is tied to the skill itself prior to being called.